index.html

```diff
@@ -1,47 +1,229 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="/css/homepage-styles.css" />
-    <link
-      href="https://fonts.googleapis.com/icon?family=Material+Icons"
-      rel="stylesheet"
-    />
-
-    <title>homepage</title>
-  </head>
-  <body>
-    <header class="header-container">
-      <img src="/images/image_geometry_1.png" alt="geometry image" />
-      <nav>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="/css/homepage-styles.css" />
```

Comment:
- [ ] It should be href="./css/homepage-styles.css"  with a dot at the beginning as well as all the images.
![review](https://user-images.githubusercontent.com/53347966/220200286-a0a358e5-cca2-4dbc-91e9-abb267c0220e.JPG)